### ISSUE TITLE
Add a "switch back to captcha" button.

### ISSUE THREAD

**Issue by dlikhten** [wjcrowcroft/MotionCAPTCHA#6]
I have friends who's dyslexia prevents them from following a line with their finger, nothing to do with training or intelligence, its a full blown bonafied disability and asking them to trace a line is pretty much an impossible task as it will take them a very very long time to do if even possible.

Its the only thing that comes to mind regarding these.

**Comment by wjcrowcroft** [comment]
Yeah, I fully agree - having a 'switch to captcha' link seems like a crucial factor in getting motionCAPTCHA to production-ready level
